src/test/scala/com/snowflake/snowpark_test/SessionSuite.scala

```diff
@@ -49,6 +49,11 @@ class SessionSuite extends SNTestBase {
     t2.run()
   }
 
+  test("Test for get or create session") {
```

Comment:
this test failed in the merge gate.
please try this
```
val session1 = Session.builder.getOrCreate
val session2 = Session.builder.getOrCreate
assert(session1 == session2)
```

Reply:
Updated!